### DISCUSSION TITLE
Import from Private Code/Module

### DISCUSSION THREAD

**Discussion by dc0d** [denoland/deno#9424] [Q&A]
What is the idiomatic way of working with private code in deno?

**Comment by kitsonk** [comment]
What do you mean by private code?

**Comment by ghost** [comment]
This is something that the service worker api [would help](https://github.com/denoland/deno/issues/5239#issuecomment-753545286) with, programmatically (#5957), besides kitsonk's [proposal](https://github.com/denoland/deno/issues/5239#issuecomment-766551288) about a json file configuration for remote files access tokens.

**Comment by axetroy** [comment]
I think this should be resolved by setup a self-host registry which provide your private code.
